setup/pi/setup-teslausb

```diff
@@ -841,6 +841,17 @@ then
   echo "source /root/bin/setup-teslausb" >> /root/.bashrc
 fi
 
+# UX courtesy reminders
+if ! grep -q remount /root/.bashrc
+then
+  echo "echo \"Use bin/remountfs_rw to allow changes.\"" >> /root/.bashrc
+fi
+
+if ! grep -q sudo /home/pi/.bashrc
+then
+  echo  "echo \"Use sudo -i to enable making changes to the system. Useful files are in /mutable/\"" >> /home/pi/.bashrc
```

Comment:
The most useful file someone might want to change is /root/teslausb_setup_variables.conf
(archiveloop.log is certainly useful and in /mutable, but it's not something you'd need to change or become root to read)